API.md

```diff
@@ -27,12 +27,15 @@ In general, you won't need to configure this middleware besides the required par
 | clientSecret        | `env.CLIENT_SECRET`             | The client secret, only required for some grants.                              |
 | clockTolerance      | `5`                             | The clock's tolerance in seconds for token verification.                       |
 | getUser             | `tokenSet => tokenSet.claims()` | An async function receiving a tokenset and returning the profile for `req.openid.user`. |
-| required            | `true`                            | If true requires authentication for all the routes in the stack. You can also provide a function to determine if is required based on the request.               |
+| required            | `true`                          | If true requires authentication for all the routes in the stack. You can also provide a function to determine if is required based on the request.               |
 | handleUnauthorizedErrors | `true`                     | Install a middleware that handles Unauthorized/401 errors by triggering the login process. |
 | routes              | `true`                          | Installs the `GET /login` and `GET /logout` route.                              |
```

Comment:
wouldn't this change with this PR as well?

Reply:
No. This switch makes authentication required on all routes when it's `true` (default but considering whether that's best) or not (using `requireAuth()`) when it's `false`.

Reply:
I mean the description of the property. If the /login and /logout paths can now be customized, maybe it's worth pointing out those new properties refer to that value?